### ISSUE TITLE
Make `btrfs filesystem mkswapfile` check if underlying filesystem has single DATA profile 

### ISSUE THREAD

**Issue by yarikoptic** [kdave/btrfs-progs#840]
ATM command might succeed but then subsequent `swapon` would fail.  Might be worth doing a check and issuing at least a warning ?

**Comment by kdave** [comment]
Yes that makes sense, the information is accessible from sysfs and not restricted. Alternatively the SEARCH_TREE ioctl can be used for that and read more information or do validation of the extents of the swapfile (e.g. the NODATASUM flag).
